.npmignore

```diff
@@ -1,7 +1,7 @@
 /*
+/dist/*
 !/dist/spectacle*
-dist/*.map
-!/docs
+/dist/*.map
```

Comment:
We were publishing `dist/bundle.js` and other example deck stuff. this hones it down more, and removes `docs` because they're not needed for downloads to save space. Current `ls`:

```
LICENSE
README.md
dist
es
lib
package.json
src
```